components/Spotlight/Spotlight.js

```diff
@@ -0,0 +1,25 @@
+import SpotlightMain from "./SpotlightMain.js";
+import SpotlightPreview from "./SpotlightPreview";
+import members from "./spotlight-members.js";
+import React from "react"; 
+
+class Spotlight extends React.Component{
+    state = {
+        // get the lastest entry from the spotlight members
+        mainMember: members[Object.keys(members).pop()],
+    };
+    selectMember = member => {
+        this.setState({mainMember: member});
+        window.scrollTo({ top: 0, behavior: 'smooth' });
+    };
+    render(){
+        return (
+            <React.Fragment>
+                <SpotlightMain member={this.state.mainMember}/>
+                <SpotlightPreview members={members} selectMember={this.selectMember}/>
+            </React.Fragment>
+        )
+    };
+}
+
```

Comment:
OH yea that makes sense. I didn't know you could have multiple components in one file, I'm guessing it would look something [like this](https://stackoverflow.com/a/50073244)? I see in this SO answer (and I think I remember writing it in my react notes) the react components are typically made 1:1 per file but like you said it logically makes sense to group them if one is just a presentational component for the main component. I agree, I think we should move `SpotlightMain` and `SpotlightPreview` to this file